src/components/EventCard/EventCard.js

```diff
@@ -89,12 +89,18 @@ function EventCard({
 	date,
 	location,
 	detailLink,
+	zoomLink,
 	imgFile,
 	disabled,
 	classes
 }) {
 	const [isHover, setIsHover] = useState(false);
 	const dateStr = moment(date).calendar();
+	const isWithin12Hours = moment.utc()
+		.isBetween(
+			moment(date).utc().subtract(12, 'hours'),
+			moment(date).utc().add(12, 'hours')
+		);
```

Comment:
In general, I'd rather us not use moment, as it's a huge library without too much of a boost in functionality. The equivalent could be done like this with the standard library, without any lack (IMO) of readability:

```js
// Outside the component.
const MINUTE = 60 * 1000;
const HOUR = 60 * MINUTE;

// Inside the component.
const isWithin12Hours = Math.abs(new Date() - date) < 12 * HOUR;
```

(Pattern stolen from [Go](https://golang.org/pkg/time/#pkg-constants).)

Reply:
I will fix this in another PR because I want to understand how this works, but at the moment don't have time. Also need to get the event list updated ASAP.

Reply:
addressing this in #118 instead. go with moment for now.